### PR TITLE
138 custom calendar feature

### LINKED ISSUE #138: Make custom calendar component for home page
**_Few pointers:_**
- This should show history also 
- Red is a period dates 
- Blue is ovulation 

image

### PR DESCRIPTION
**Pull request by dibakar95**
Issue #138 custom-Calendar-with-Header 
Brief description of what is fixed or changed
- Made a custom Calendar 
- Made Changes to HomeScreen/index.js

Put an x in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation update (Documentation content changed)
- [ ]  Other (please describe):
Checklist
Put an x in the boxes that apply

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by sarthology — DISMISSED** [review]

File: src/Components/Calendar/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        headerData={{
+          calendarDate: calendarDate.format('MMM, YYYY'),
+        }}
+        style={styles.calendarStyle}
+        theme={{
+          textDayFontSize: 18,
+        }}
+        markingType={'custom'} //{"period"}
+        markedDates={{
+          // [this.state.calendarDate]: {
```
Comment (narender2031) [inline src/Components/Calendar/index.js]: Remove the commented code

**Review by narender2031 — CHANGES_REQUESTED** [review]

**Comment by dibakar95** [comment]
@narender2031  sir please check it now.

File: src/Components/Calendar/index.js (hunk 1 above)
Comment (dibakar95) [reply src/Components/Calendar/index.js]: Some methods do not have a body but it was a must to define them. And day component may be needed in future. For that purpose I let it remain commented.

**Review by narender2031 — APPROVED** [review]

**Comment by dibakar95** [comment]
@sarthology please check now.

**Review by sarthology — APPROVED** [review]

**Pull request merged**